index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/4.4.1/cosmo/bootstrap.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css"
+    />
+    <link rel="stylesheet" href="./css/custom.css" />
+    <title>Tic Tac Toe</title>
+  </head>
+  <body>
+    <div class="container">
+      <h1 class="text-center font-weight-bold">Tic-Tac-Toe</h1>
+      <!-- Players Names  -->
+      <form class="m-auto w-50" id="game-players-form">
+        <div class="form-group">
+          <label for="playerXLabel">First Player Name : </label>
+          <input type="text" required class="form-control" id="playerX" />
+        </div>
+        <div class="form-group">
+          <label for="playerOLabel">Second Player Name : </label>
+          <input type="text" class="form-control" id="playerO" />
+        </div>
+        <div class="col text-center">
+          <button
+            type="button"
+            class="btn btn-primary text-center"
+            id="start-game"
+            onclick="game.start()"
```

Comment:
It is not a good practice to mix javascript with HTML.
You have already added the id, so on your js file
```javascript
document.querySelector('#start-game').addEventListener('click', () => {
game.start();
});
```